frontend/packages/kubevirt-plugin/src/plugin.tsx

```diff
@@ -191,6 +195,28 @@ const plugin: Plugin<ConsumedExtensions> = [
       required: FLAG_KUBEVIRT,
     },
   },
+  {
+    type: 'Dashboards/Storage/TopConsumers/Used',
+    properties: {
+      name: 'VMs',
+      queryName: 'VMS_BY_USED',
```

Comment:
we should not need to specify this `queryName` at all. Why do we even need it ? From extension type `type: 'Dashboards/Storage/TopConsumers/Used',` you know that this is for Top Consumers Used and `name` prop should be unique for given type of extension. So in code you could just do `${name}_BY_USED` right ?